segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/SegmentRollingPolicy.java

```diff
@@ -16,7 +16,10 @@
  * A generic rolling policy that can be applied to any Storage unit.
  */
 public final class SegmentRollingPolicy {
-    public static final SegmentRollingPolicy NO_ROLLING = new SegmentRollingPolicy(Long.MAX_VALUE);
+    /**
+     * Max rolling length is 2^62 so that we can use CompactLong in serialization everywhere.
```

Comment:
This comment needs to change similarly to the comment in the previous PR, it should be `2^61 - 1` instead of `2^62`. I'm actually wondering why I'm seeing this here, is it because this PR includes changes of Part 1?

Reply:
I actually confirmed that this is the case, that the changes of Part 1 are also here. What's the plan exactly? Do you expect git to automatically sort this out when Part 1 is merged?

Reply:
Re-targeted base branch.